### ISSUE TITLE
Microsoft.Data.Services.Client version problem for Azure Storage Queues

### ISSUE THREAD

**Issue by dvdstelt** [Particular/ServiceControl#468]
I followed the manual
http://docs.particular.net/servicecontrol/multi-transport-support

But with the latest version of ServiceControl, the Microsoft.Data.Services.Client has an incorrect version. I downgraded the dll and it's working again. Probably good to update the manual with the correct packages. Probably best to make the entire experience a lot smoother :-)

**Comment by gbiellem** [comment]
@dvdstelt  - we are working on making the installer smoother for other transports. In the meantime I'll check this out

**Comment by gbiellem** [comment]
@dvdstelt  - Thanks for the heads up. 

It looks like I did it in 1.5.1 when trying to fix a nuget inconsistency between our test suite and ServiceControl.  :(

Rather than downgrade you can add a assembly redirect.  I'll update the manual to reflect this.  Here a sample config with the redirect:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <connectionStrings>
        <add name="NServiceBus/Transport" connectionString="UseDevelopmentStorage=true;" />
    </connectionStrings>
    <appSettings>
        <add key="ServiceControl/ForwardAuditMessages" value="true" />
        <add key="ServiceControl/TransportType" value="NServiceBus.AzureStorageQueue, NServiceBus.Azure.Transports.WindowsAzureStorageQueues" />
    </appSettings>
    <runtime>
        <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
            <dependentAssembly>
                <assemblyIdentity name="Microsoft.Data.Services.Client" publicKeyToken="31bf3856ad364e35"
                    culture="neutral" />
                <bindingRedirect oldVersion="0.0.0.0-5.6.3.0" newVersion="5.6.3.0" />
            </dependentAssembly>
        </assemblyBinding>
    </runtime>
</configuration>
```
I'll add this to the documentation for now

**Comment by johnsimons** [comment]
Fixed as part of #480
